Automattic/wp-calypso
# Framework: Remove obsolete lib/md5

Seems like in #11261 we introduced an empty `client/lib/md5/index.jsx` file - probably because of a bad rebase. It's not used at all and can be removed.

To test:
* Checkout this branch
* Verify this is indeed not used
